test/tests/visibility/visibility_override.c

```diff
@@ -12,5 +12,5 @@ char *core_version = "This is a new variable";
 
 int main(void)
 {
-	return strcmp(core_version, ROOTSIM_VERSION) == 0;
+	return strcmp(core_version, "3.0.0-alpha.3") == 0;
```

Comment:
The philosophy here is that tests should be written as a user of the library (e.g.: they shouldn't rely on compile time definitions fed during the build of the ROOT-Sim core library). As a result of that, inappropriate changes to the version string will be detected by this test. Of course, the disadvantage would be the need to manually update the string at each version bump.

Reply:
To be fair, in this test, the ```strcmp()``` check should be done against the overridden value of the version string, i.e.: ```"This is a new variable"```.

Reply:
Ok, we can pay the minimal "cost" to change a string once the test will fail because we forget this after the bump  😁

So the other test should be corrected, right?

Reply:
I improved this test with the comparison against the overridden string. The other test _should_ remain the same, if we agree on the philosophy.

Reply:
Agreed!